src/components/templates/PosterPage/components/ShareModal/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { FACEBOOK_APP_ID } from "secrets";
+import { PosterPageVenue } from "types/venues";
+import { WithId } from "utils/id";
+
+export const getTitleTextForSharing = (
+  venue: WithId<PosterPageVenue>,
+  url: string
+) => {
+  if (!venue.poster) return "Check out this OHBM Poster";
+
+  return `Check out this OHBM Poster, ${venue.poster.title} by ${venue.poster.authorName} at ${url}`;
+};
+
+export const getFacebookHref = (url: string, text: string) =>
+  `https://www.facebook.com/sharer/sharer.php?app_id=${FACEBOOK_APP_ID}&u=${url}&quote=${text}&hashtag=#OHBM2021`;
+
+export const getTwitterHref = (text: string) =>
+  `https://twitter.com/intent/tweet?text=${text}&hashtags=OHBM2021`;
```

Comment:
As I called out in a previous review, this contains hardcoded text related to OHBM, which needs to be removed/changed before this PR can be merged. This feature needs to be generic.

(@mike-lvov FYI)

Reply:
Currently I leave it here just for test.

Reply:
What do you mean leave it here for test?

This needs to be loaded from settings/sovereign venue config/similar, not hard coded.

Reply:
Done in prev commits